packages/react-reconciler/src/ReactStrictModeWarnings.js

```diff
@@ -186,12 +203,19 @@ if (__DEV__) {
 
       lowPriorityWarning(
         false,
-        'componentWillUpdate is deprecated and will be removed in the next major version. ' +
-          'Use componentDidUpdate instead. As a temporary workaround, ' +
-          'you can rename to UNSAFE_componentWillUpdate.' +
-          '\n\nPlease update the following components: %s' +
-          '\n\nLearn more about this warning here:' +
-          '\nhttps://fb.me/react-async-component-lifecycle-hooks',
+        'componentWillUpdate has been renamed to UNSAFE_componentWillUpdate, ' +
+          "and the old name won't work in the next major version of React.\n" +
+          'We suggest doing one of the following:\n' +
+          '- If you fetch data or perform other side effects in componentWillUpdate, ' +
+          'move this logic into componentDidUpdate.\n' +
+          "- If you're reading DOM properties before an update, " +
```

Comment:
Legit use cases for `getSnapshotBeforeUpdate` seem so uncommon, I wonder if it's worth mentioning here at all? I don't have a strong preference one way or another, but a slight preference to maybe not mention it here for fear of encouraging bad patterns for those maybe lacking context of when the method should be used.

Reply:
I think I'll leave it here. We seem fairly comprehensive for the others, it would be frustrating if the dev had to go to a doc to discover a new api for just this one case.

Reply:
I guess I'm concerned that if people just move stuff to that method, without the added context of _why_ it exists, they might be tempted to do mutations/side effects in it.

Reply:
They would probably try componentDidUpdate before that though? I dunno tbh. I could remove this, but seems like we should use same reasoning to remove getDerivedStateFromProps if so.